src/internal/operations/restore.go

```diff
@@ -35,11 +35,11 @@ import (
 type RestoreOperation struct {
 	operation
 
-	BackupID    model.StableID             `json:"backupID"`
-	Destination control.RestoreDestination `json:"destination"`
-	Results     RestoreResults             `json:"results"`
-	Selectors   selectors.Selector         `json:"selectors"`
-	Version     string                     `json:"version"`
+	BackupID   model.StableID
```

Comment:
Curious, why did we remove struct tags for `RestoreOperation`?

Reply:
We don't serialize the model.  The declaration was useless.